src/common/constants.ts

```diff
@@ -25,10 +27,14 @@ export const domains = [
     'deriv-com-git-fork',
 ] //deriv-com-git-fork for vercel server, localhost - for developer mode
 
-const getDomainUrl = (): string =>
-    isBrowser() && window.location.hostname in domain_url_pair
-        ? domain_url_pair[window.location.hostname]
+const getDomainUrl = (): string => {
+    const staging_flag = isBrowser() ? localStorage.getItem('staging_demo') : null
```

Comment:
[Clarification] This is solely for QA testing purposes?

Reply:
yes